examples/named_cluster/main.tf

```diff
@@ -77,4 +77,9 @@ module "aks_cluster_name" {
   rbac_aad                          = true
   rbac_aad_managed                  = true
   role_based_access_control_enabled = true
+
+  # KMS encrption
+  key_vault_kms_enabled = true
+  key_vault_kms_key_id  = azurerm_key_vault_key.kms.id
```

Comment:
This should be `kms_key_vault_key_id` according to your change in `variable.tf`

Reply:
Also copy pasta error before refacotoring :/